src/pages/health-declaration/health-declaration.ts

```diff
@@ -173,6 +186,14 @@ export class HealthDeclarationPage extends BasePageComponent {
       ),
     );
     this.subscription = merged$.subscribe();
+    this.licenseProvidedSubscription = this.pageState.licenseProvided$.
```

Comment:
Is there are reason we are creating new subscriptions instead of merging them into the merged$ subscription

Reply:
I hadn't realised the purpose of this merged subscription.  Have incorporated the logic into this subscription. Lot cleaner. Thanks.